.github/workflows/ci.yml

```diff
@@ -10,7 +10,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["2.5", "2.6", "2.7", "3.0", "3.1", "3.2", "3.3", ruby-head, jruby-9.2, jruby-9.3, jruby-head]
+        ruby: ["2.5", "2.6", "2.7", "3.0", "3.1", "3.2", "3.3", "3.4", ruby-head, jruby-9.3, jruby-head]
+        include:
+          - ruby: jruby-9.2
+            runs-on: ubuntu-20.04
```

Comment:
I think you might have to use a variable in `jobs.test.runs-on`, though I don't recall if GHA allows this.

Reply:
Yeah, I think `runs-on: ${{ matrix.runs-on }}` should do it. ([GitHub suggests calling the variable `os`](https://runs-on.com/github-actions/the-matrix-strategy/).)

Reply:
Nice, we are in sync, I just read this the moment after doing that.

Reply:
Aha, the Annotation from GitHub Actions tells me it's time to ditch JRuby 9.2 from the CI matrix. I'll do that just now.

> This is a scheduled Ubuntu 20.04 retirement. Ubuntu 20.04 LTS runner will be removed on 2025-04-15. For more details, see https://github.com/actions/runner-images/issues/11101

Reply:
Yeah, let's ditch it.